frontend/app/utils/env.server.ts

```diff
@@ -22,7 +22,7 @@ function tryOrElseFalse(fn: () => unknown) {
 const validMockNames = ['cct', 'lookup', 'power-platform', 'raoidc', 'status-check', 'wsaddress', 'subscription'] as const;
 export type MockName = (typeof validMockNames)[number];
 
-const validFeatureNames = ['doc-upload', 'email-alerts', 'hcaptcha', 'view-personal-info', 'view-applications', 'view-letters', 'view-messages', 'edit-personal-info', 'status', 'authenticated-status-check'] as const;
+const validFeatureNames = ['doc-upload', 'email-alerts', 'hcaptcha', 'view-personal-info', 'view-applications', 'view-letters', 'view-messages', 'edit-personal-info', 'status', 'authenticated-status-check', 'show-banner'] as const;
```

Comment:
Change it to a more specific name `show-prototype-banner`

Reply:
That's the thing, we might want to use it for any other reason. Possibly?

Reply:
That feature flag name doesn't reflect the feature that is showing a prototype banner for the moment.